src/Tags/ResponsiveTag.php

```diff
@@ -92,4 +92,26 @@ private function getAttributeString(): string
                 return $name . '="' . $value . '"';
             })->implode(' ');
     }
+
+    private function includePlaceholder(): bool
+    {
+        $includePlaceholder = $this->params->get('placeholder');
+
+        if (isset($includePlaceholder)) {
+            return $includePlaceholder;
+        }
+
+        return config('statamic.responsive-images.placeholder');
+    }
+
+    private function includeWebp(): bool
+    {
+        $includeWebp = $this->params->get('webp');
+
+        if (isset($includeWebp)) {
```

Comment:
Same here, this will always return true, since you just set the variable above, better to use `is_null`

Reply:
Same here.

Reply:
Yes, but if you do `$includeWebp = null`, the `isset` call would still return `true`

Reply:
In what scenario would `$includeWebp` be `null`? Even if you do `placeholder=""` on the tag, it will be an empty string, not null.

Reply:
You could do `if ($this->params->has('webp'))` instead as a check

Reply:
Good one! I will change that.